requirements.ci.txt

```diff
@@ -1,7 +1,10 @@
 codecov
-flake8
+flake8==4.0.1
```

Comment:
[question]
What is the usage of the requirements files when we have environment ones?

Reply:
This allows the user to choose whether to install with pip or conda.